### PR TITLE
Feature/pl 2204 add lokalise download action

### PR DESCRIPTION
**Pull request by mrden1**

### CODE REVIEW

File: packages/lokalise-download/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+|       trigger       | Trigger integration exports (must be enabled in project settings) |
+
+> **_Detailed information_**: `lokalise2 file download --help`
+
+<br>
+
+### Usage
+
+```yaml
+name: Lokilise sync
```
Comment (Rendalf) [inline packages/lokalise-download/README.md:25]: typo: Lokalise

File: packages/lokalise-download/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+          lokalise-project-id: '<project_id>'
+          file-path: './src/locales'
+          format: json
+          bundle-structure: 'src/locales/%LANG_ISO%'
+          placeholder-format: 'i18n'
+          trigger: 'github'
+          add_newline_eof: true
+          filter_repositories: 'websome'
+          filter_filenames: '<filter_filename>'
+          plural_format: ''icu
```
Comment (Rendalf) [inline packages/lokalise-download/README.md:56]: typo? `''icu` -> `'icu'`?

File: packages/lokalise-download/action.yml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    description: 'Override the default plural format for the file type'
+    required: true
+    default: 'icu'
+  indentation:
+    description: 'Provide to override default indentation in supported files'
+    required: true
+    default: '2sp'
+  export_sort:
+    description: 'Allowed value are first_added, last_added, last_updated, a_z, z_a'
+    required: true
```
Comment (Rendalf) [inline packages/lokalise-download/action.yml:52]: does it make sense to make it required? can we just put it as `a_z` by default?
the same of indendation, plural_format, eof, trigger, bundle structure etc

Comment (mrden1) [reply packages/lokalise-download/action.yml:53]: added default values

**Pull request merged**